docs/source/models/intervention_models/intrapartum/misoprostol_intervention.rst

```diff
@@ -0,0 +1,268 @@
+.. _misoprostol_intervention:
+
+============================================
+Misoprostol for treating maternal hemorrhage
+============================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. list-table:: Abbreviations
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Abbreviation
+    - Definition
+    - Note
+  * - BEmONC
+    - Basic emergency obstetric and neonatal care
+    - Operationalized as facilities without C-section capabilities
+  * - CEmONC
+    - Comprehensive emergency obstetric and neonatal care
+    - Operationalized as facilities with capabilities to perform  C-section
+  * - PAF
+    - Population Attributable Fraction
+    - 
+  * - RR
+    - Relative Risk
+    - 
+  * - ANC
+    - Antenatal Care 
+    - 
+  * - PPH
+    - Postpartum Hemorrhage
+    - For consistency with the official GBD Cause list and our :ref:`Maternal hemorrhage cause model <2021_cause_maternal_hemorrhage_mncnh>`,
+      we use the term 'maternal hemorrhage' instead of PPH throughout the course of this document.
+
+Intervention Overview
+-----------------------
+
+Misoprostol is an intramuscular prophylactic prostaglandin that can be taken orally or sublingually during labor to prevent or reduce incidence of maternal hemorrhage. Because 
```

Comment:
```suggestion
Misoprostol is a prophylactic prostaglandin that can be taken orally or sublingually during labor to prevent or reduce incidence of maternal hemorrhage. Because 
```

I think it's a bit confusing to have "intramuscular" here given that the intervention as we are modeling it is administered orally (or vaginally, I believe!)